ide-config/intellij/Brandwatch.xml

```diff
@@ -67,5 +67,10 @@
   </editorconfig>
   <codeStyleSettings language="JAVA">
     <option name="RIGHT_MARGIN" value="140" />
+    <option name="ALIGN_MULTILINE_CHAINED_METHODS" value="true" />
```

Comment:
Is this a mandatory alignment or simply enabling the option? If mandatory, it's something I'm not a huge fan of being forced to do.

Reply:
not mandatory just based of @spmason having it set, I don't dislike it but any objection about enabling it is good enough for me to remove it from this PR at least :D